src/scaffolding/master.scaffolding.hbs

```diff
@@ -145,26 +145,26 @@
     <div class="row row--reducedWidth">
         <div class="col">
                 <bio-section-headline inverted data-resources="[{paths: ['components/BioSectionHeadline/index.js']}]">
-                    You will </br> <strong>Benefit</strong>
+                    Develop digital experiences </br> <strong>Flexible. Constant. Across All Devices.</strong>
                 </bio-section-headline>
             <div class="extendedBenefits__container">
                 <bio-benefit inverted data-resources="[{paths: ['components/BioBenefit/index.js']}]"
-                    imgsrc="../../_assets/free.svg" title="Keep focus">
-                    You develop high quality components, biotope takes care of architecture and dev environment for you
+                    imgsrc="../../_assets/icon-quality-focus.svg" title="Quality Focus">
+                    Develop high quality components instead of the architecture and development environment required. That's Biotope's job.
                 </bio-benefit>
                 <bio-benefit inverted data-resources="[{paths: ['components/BioBenefit/index.js']}]"
-                    imgsrc="../../_assets/free.svg" title="Open source">
-                    Keep all your design systems in one central place for multiple projects.
+                    imgsrc="../../_assets/icon-consistent-design.svg" title="Consistent Design">
+                    Roll out your interface from a single point of truth - simultaneously for all of your digital products. Keep your brand identity consistent across all your websites, apps or IoT say goodbye to design debt.
```

Comment:
```suggestion
                    Roll out your interface from a single point of truth - simultaneously for all of your digital products. Keep your brand identity consistent across all your websites, apps or IoT. Say goodbye to design debt.
```

Shouldn't these phrases be separated in some way?

Reply:
No you have written in your template <p><slot></slot></p> so I guess it is fine